src/Microsoft.Restier.Core/Microsoft.Restier.Core.csproj

```diff
@@ -8,7 +8,7 @@
     <RootNamespace>Microsoft.Restier.Core</RootNamespace>
     <AssemblyName>Microsoft.Restier.Core</AssemblyName>
     <DocumentationFile>$(OutputPath)$(AssemblyName).xml</DocumentationFile>
-    <RunCodeAnalysis>$(CodeAnalysis)</RunCodeAnalysis>
+    <RunCodeAnalysis>true</RunCodeAnalysis>
```

Comment:
Per discussion with Biao, I am considering changing this line to

<RunCodeAnalysis Condition="'$(RunCodeAnalysis)' == ''">true</RunCodeAnalysis>